modules/hal_nordic/nrfs/backends/nrfs_backend_ipc_service.c

```diff
@@ -41,18 +41,17 @@ struct ipc_channel_config {
 
 static struct ipc_ept_cfg ipc_sysctrl_ept_cfg = {
 	.name = "ipc_to_sysctrl",
-	.cb = {
-		.bound    = ipc_sysctrl_ept_bound,
-		.received = ipc_sysctrl_ept_recv,
-	},
+	.cb =	{
+			.bound = ipc_sysctrl_ept_bound,
+			.received = ipc_sysctrl_ept_recv,
+		},
```

Comment:
What is the point in using such weird alignment?
Anyway, all the formatting changes (if really needed) should be done in a separate commit to not obscure the actual changes in the error handling.